content/logs/_index.md

```diff
@@ -2,320 +2,178 @@
+| **Source**  | This corresponds to the integration name: the technology or service that originated to log. When it matches ab integration name, Datadog automatically installs the corresponding parsers and facets. |
+| **Service** | This is the name of the application or service generating the log events. It is used to switch from Logs to APMso make sure you define the same value if you use both products.                       |
+| **Message** | By default, Datadog ingests the value of message as the body of the log entry. That value is then highlighted and displayed in the logstream, where it is indexed for full text search.               |
 
-## Stream logs through TCP/UDP
-Set `type` to **tcp** or **udp** depending of your protocol then specify the `port` of your incoming connection.
+Your logs are now collected and centralized into the [Log Explorer][17] view, but your journey doesn’t ends here. 
 
-Example:
-If your PHP application does not log to a file, but instead forwards its logs via TCP, create a configuration file that specifies the port to receive as in the example below:
+{{< img src="logs/log_explorer_view.png" alt="Log explorer view" responsive="true" popup="true">}}
 
-```yaml
-init_config:
-instances:
+## Search your Logs
 
-##Log section
-logs:
-  - type: tcp
-    port: 10518
-    service: webapp
-    source: php
-    sourcecategory: front
+Use integration [facets][18] or [add custom ones][19] to slice and dice in your logs. You can also use tags shared by logs, metrics and traces to filter your data or even [free text search][20] on the log message:
 
-```
-* [Restart your Agent][3]
+{{< img src="logs/search_your_logs.gif" alt="Search your logs" responsive="true" popup="true">}}
 
-The Agent supports raw string, JSON and Syslog formated logs. If you are sending logs in batch, use break line characters to separate your logs.
+Follow our [guide to explore your logs][17] for a more detailed explanation of all the Log Explorer features include wildcards usage and query on numerical values.
 
-## Advanced log collection functions
+## Graph and Analytics
 
-### Filter logs
+Now that your logs are parsed and that you have facets and measure over the wanted attributes graph log queries and see maximum, averages, percentiles, unique counts and more.
 
-All logs are not equal and you may want to send only a specific subset of logs to Datadog.
-To achieve this use the `log_processing_rules` parameter in your configuration file with the **exclude_at_match** or **include_at_match** `type`.
+1. Choose a [Measure][21] or [Facet][18] to graph. [A Measure][21] lets you choose the aggregation function whereas a [Facet][18] displays the unique count.
+2. Select the aggregation function for the [Measure][21] you want to graph:
+3. Use Tag or [Facet][18] to split your graph.
+4. Choose to display either the X top or bottom values according to the selected [measure][21].
```

Comment:
Seems that we use `Measure` and `measure`. No hard feeling on it but maybe we should either always or never capitalise it.